### ISSUE TITLE
Fragpipe processes returning nonzero exit codes

### ISSUE THREAD

**Issue by jess-y-li** [Nesvilab/FragPipe#954]
I have been trying to process raw DDA IM-MS tims-TOF LC-MS files using Fragpipe. The experiment I did involved two conditions, control and experimental. I did two biological replicates, with six fractions injected for each replicate, for a total of 24 raw files to process (12 per replicate). I had success in the past when I ran these files by replicate (ran each set of 12 files separately so that the output folder only contained data for each replicate).
I want to compare the findings between the two replicates, so I tried running again with all 24 raw data files with Match Between Runs (MBR) selected. I have tried probably 10 times now, and every time there seems to be a different nonzero exit code being returned, resulting in the cancelling of x number of remaining tasks. I will upload the file for the most recent error, where it specifies "Process 'MSBooster' finished, exit code: 1'.
I have also had "Process 'SpecLibGen' finished, exit code: 1', in which case I unchecked the "Generate spectral library from search results" option. 
Running again showed "Process 'IonQuant' finished, exit code:1". 
At this point, I have tried updating all of the tools (MSFragger, IonQuant, Philosopher, Python). I have tried running it on a different computer. I have tried running on both FragPipe v18.0 and v19.0. I have tried inputting my 24 raw files as one replicate. I have turned off certain features when running, including the MBR function. I am not sure what the issue is. Again, the data file uploaded is the most recent attempt, but I have been receiving different error messages. I would really appreciate if anyone has insight into this matter.

[log_2023-01-05_13-15-19.txt](https://github.com/Nesvilab/FragPipe/files/10355199/log_2023-01-05_13-15-19.txt)

**Comment by fcyu** [comment]
The FragPipe version you are using is not compatible with the MSFragger 3.6. Can you upgrade FragPipe and try again?

Thanks,

Fengchao

**Comment by jess-y-li** [comment]
Hi Fengchao,

I actually started running on FragPipe 19.0, which is the most recent
version available to my knowledge. It returned several errors so I thought
maybe it was the new version, as I had no troubles with the old version.
Following that, I went back to 18.0 on a different computer.
Your advice explains the errors I am receiving on the log I uploaded, but
not the errors in my previous tries. Would it be more helpful for me to try
running again on 19.0 and upload the error log for that?

Thank you,
Jessica
